blocks/library/nextpage/index.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { RawHTML } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+import { createBlock } from '../../api';
+
+export const name = 'core/nextpage';
+
+export const settings = {
+	title: __( 'Page break' ),
+
+	description: __( 'This block allows you to set break points on your post. Visitors of your blog are then presented with content split into multiple pages.' ),
+
```

Comment:
A `keyword` property here would be useful to include things like `next page`, helping with discovery in the block inserters.